src/SFA.DAS.EAS.Portal.UnitTests/Portal/Application/Services/AccountDocumentServicesTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.CosmosDb;
+using SFA.DAS.EAS.Portal.Application.Services;
+using SFA.DAS.EAS.Portal.Client.Database.Models;
+using SFA.DAS.EAS.Portal.Client.Types;
+using SFA.DAS.EAS.Portal.Database;
+using SFA.DAS.EAS.Portal.UnitTests.Builders;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions.Common;
+using Microsoft.ApplicationInsights.Metrics.Extensibility;
+using Microsoft.Azure.Documents.Client;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Portal.Application.Services
+{
+    [Parallelizable]
+    [TestFixture]
+    class AccountDocumentServicesTest
+    {
+        public class TestContext
```

Comment:
What purpose does the test context serve here? all this logic could be done it SetUp and therefore no need for the extra class with a bunch of public properties

Reply:
It's the same way the other tests are structured, so I just followed the pattern, happy to switch if preferred